guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+      modalImage: cashu-id3@2x
+      alt: Mint contact information display in a wallet interface
+      caption: Contact information is optional for the mint operator to include. If there is no contact information, consider not displaying this metadata.
+    - file: cashu-id4
+      modalImage: cashu-id4@2x
+      alt: Example of important announcements display in a wallet interface
+      caption: A message of the day can consists of anything the mint operator wants to tell the user. It can for example be used to announce a new feature or a upcoming maintenance.
+    - file: cashu-id5
+      modalImage: cashu-id5@2x
+      alt: Example of supported features display in a wallet interface
+      caption: Supported features are pulled by indexing a mints information and reading what nuts the mint has implemented.
+
+---
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/best-practicies.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/best-practicies@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/best-practicies-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/best-practicies-mobile@2x.jpg"
+   alt-text = "ecash best practices illustration"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+<!--
+
+
+This is a summary page of best design practices for both bitcoin-backed ecash protocols: Cashu and Fedimint.
+
+The design source for screen mock-ups on this page can be found here:
+ADD Figma URL
+
+
+-->
+
+# Ecash design best practices
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+## General Ecash Best Practices
+When designing bitcoin-backed ecash applications it's important to prioritize clear and intuitive interfaces that allow users to easily manage their tokens, whether they are minting, sending, or redeeming them.
+
+### Multiple Mint or Federation Display
+{% include image-gallery.html pages = page.images_mint-list %}
+
+When displaying multiple mints or federations in a bitcoin-backed ecash application, there are several best practices to consider:
+
+1. **Clear Hierarchy and Organization**: Ensure that mints or federations are organized in a way that is easy for users to navigate. Group mints by relevance, trust level, or frequency of use, and consider using sorting or filtering options to help users find the right mint or federation quickly.
+
+2. **Prominent Metadata Display**: Use metadata fields like mint name, icon, URL, and description, to give users insight into each mint or federations identity. Display this information prominently, so users can easily verify the authenticity and relevance of each mint or federation.
+
+3. **Quick Mint Access**: Provide easy access to mint or federation details directly from the mint or federation list. This allows users to interact with mints or federations more fluidly without needing to navigate through multiple screens.
+
+#### Pending Tokens
```

Comment:
This note on terminology from the transaction page is helpful:
`In most places in the Guide, the term “transaction” and “payment” are used interchangeably. For the remainder of this page, the term transaction refers specifically to on-chain bitcoin transactions, and the term payment refers specifically to payments routed over the lightning network.`

Since ecash is closer to the user experience and use case of lightning, we should use the term ‘payments.’ I went back and updated ‘transactions’ with ‘payments’ where it made sense.